orders/views.py

```diff
@@ -1,3 +1,33 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
\ No newline at end of file
+from django.http      import JsonResponse
+from django.views     import View
+
+from orders.models    import Order, OrderStatus
+from spaces.models    import Option
+from users.utils      import login_decorator
+
+class OrderView(View):
+    @login_decorator
+    def post(self, request):
+        try:
+            space_id = request.GET.get("space_id", None)
+            data     = json.loads(request.body)
+                 
+            if Order.objects.filter(user_id = request.user.id, status_id = OrderStatus.Status.WAITING.value).exists():
+                Order.objects.filter(user_id = request.user.id, status_id = OrderStatus.Status.WAITING.value).delete()
+
+            Order.objects.create(
+                space_id  = space_id,
+                user_id   = request.user.id, 
+                status_id = OrderStatus.Status.WAITING.value, 
+                date      = data['date'],
+                count     = data['count'],
+                option    = Option.objects.get(space_id = space_id, option=data['option']),
```

Comment:
option_id의 존재 유무를 판단 후에 get 대신에 option_id를 직접 넣어줘도 될 것 같아요.
```suggestion
                option_id    = data['option'],
```

Reply:
option_id를 받기로 했습니다!

Reply:
예약 관련 GET Method까지 추가하여 올렸습니다!